doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2965,6 +2965,8 @@ references to automatically generated names.
 Performance-oriented tactic variants
 ------------------------------------
 
+.. todo PR: move the following adjacent to the `change` tactic in the rewriting chapter?
```

Comment:
Yes, I think it would make sense.

Reply:
I added this where `change` currently is.  Awaiting your recommendation where to put `change`.